client/question/components/LearnerViewDirective.js

```diff
@@ -115,7 +121,7 @@ tie.directive('learnerView', [function() {
                 <li class="tie-leave-feedback-button"
                     title="Click to leave your feedback about TIE">
                   <a href="https://goo.gl/CcfCq4" target="_blank">
-                    Leave Feedback
+                    Leave feedback about TIE
```

Comment:
I think I'd use "on" rather than "about", but I'm happy to defer if you think "about" is better.